j2k/testData/fileOrElement/methodCallExpression/stringMethods.kt

```diff
@@ -7,7 +7,7 @@ internal class A {
     @Throws(Exception::class)
     fun constructors() {
         String()
-        // TODO: new String("original");
+        "original"
```

Comment:
@piotrek1543 If you can suggest a different way to fix the underlying issue so that J2K does not generate red code, you're welcome to do so. Leaving a comment saying that you're unsure whether something is the right fix is not helpful; please don't.